src/fidesops/core/config.py

```diff
@@ -116,6 +116,7 @@ class SecuritySettings(FidesSettings):
 
     AES_ENCRYPTION_KEY_LENGTH: int = 16
     AES_GCM_NONCE_LENGTH: int = 12
+    DEFAULT_ENCRYPTION_BYTE_LENGTH: int = 32
```

Comment:
Copied over from Python secrets module default. Question is - should we have a configurable byte length per encryption strategy and secret type? (like we do for `AES_ENCRYPTION_KEY_LENGTH`).

Reply:
TBH I suspect these config values are really just constants - I don't believe you can modify the AES key or nonce length, for example... similarly, I think the byte length you're adding here also probably *has* to be 32. If none of these values are configurable, I'd prefer we move these out of the config class and declare them as constants in the encryption source files instead.

Reply:
it's configurable, and byte length can be any specified length. If not provided, the default package val is 32. For now, I'll just declare it as a const in the encryption level file